WordPress/src/main/java/org/wordpress/android/ui/bloggingreminders/BloggingRemindersViewModel.kt

```diff
@@ -42,15 +44,17 @@ class BloggingRemindersViewModel @Inject constructor(
     private val resourceProvider: ResourceProvider,
     private val prologueBuilder: PrologueBuilder,
     private val daySelectionBuilder: DaySelectionBuilder,
-    private val dayLabelUtils: DayLabelUtils
+    private val dayLabelUtils: DayLabelUtils,
+    private val analyticsTracker: BloggingRemindersAnalyticsTracker
 ) : ScopedViewModel(mainDispatcher) {
     private val _isBottomSheetShowing = MutableLiveData<Event<Boolean>>()
     val isBottomSheetShowing = _isBottomSheetShowing as LiveData<Event<Boolean>>
     private val _selectedScreen = MutableLiveData<Screen>()
+    private val selectedScreen = _selectedScreen.perform { onScreenChanged(it) }
```

Comment:
I think you could inline this, right?

Reply:
Hmm, I'm not sure how that would work 🤔

`_selectedScreen` is a `MutableLiveData` that has its value set by other methods inside the view model, while `selectedScreen` is a `LiveData` that performs a side effect action whenever that value changes. 

To inline these, I believe we would need to have a single `MutableLiveData` that is capable of handling side effects whenever its _own_ value changes. It's possible I'm missing something here, but I just can't see how we could do that – maybe creating our own `MutableLiveData` subclass, but I guess that's not what you had in mind...

Anyway, I'm gonna merge this for now, but please let me know if I'm missing something and then I'll address that in a separate PR.